src/main/java/frc/robot/subsystems/MAXSwerveModule.java

```diff
@@ -38,7 +43,12 @@
  *
  * The module can be purchased here: https://www.revrobotics.com/rev-21-3005/
  */
-public class MAXSwerveModule extends SubsystemBase {
+public class MAXSwerveModule {
+  private final int kPOSSlot = 0;
```

Comment:
Add a comment above these to describe what they are. Someone viewing this in a couple of weeks, months, or years will not know at all what these mean.

Additionally, because they are not enums, they should be set in the constructor like everything else!